clients/cli/src/setup.rs

```diff
@@ -166,7 +173,7 @@ pub fn clear_node_id() -> std::io::Result<()> {
     }
 
     // if the nexus directory exists, check if the node-id file exists
-    let node_id_path = home_path.join(".nexus").join("node-id");
+    let node_id_path = home_path.join(".nexus").join("config.json");
```

Comment:
config name here again

Reply:
is this fixed and you missed the push?

Reply:
no good catch, I thought you were referring to a different place that I had fixed